.github/workflows/code.yml

```diff
@@ -139,7 +139,7 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: dist
-          path: dist/*
+          path: dist
```

Comment:
I assume this is working? I had to add /* earlier but maybe that was when we had subfolders?

Reply:
Seems to be working, There is a single artifact called `dist` that contains the wheel, sdist and `requirements*.txt`:
https://github.com/DiamondLightSource/python3-pip-skeleton-cli/suites/8261099796/artifacts/361392560